src/index.js

```diff
@@ -0,0 +1,4 @@
+// eslint-disable-next-line quotes
```

Comment:
- [ ] This eslint disable can be fixed. Make the required change and ensure not to disable any eslint rules